benchmarks/cases/mpl_redraw.py

```diff
@@ -0,0 +1,30 @@
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import tempfile
+
+
+# No need for anything other than the agg backend, and we don't want
+# windows popping up as we are running these tests.
+plt.switch_backend('agg')
+
+
+def create_pc_png():
+    fig = plt.figure()
+    ax = fig.add_subplot(1, 1, 1, projection=ccrs.PlateCarree())
+    ax.coastlines()
+    ax.stock_img()
+    with tempfile.NamedTemporaryFile(suffix=".png") as fh:
+        plt.savefig(fh.name)
+    plt.close(fig)
+
+
+def time_draw_speed():
+    create_pc_png()
+
+
+def time_redraw():
+    # Successive re-draws could have various caching
```

Comment:
But you've closed the figure, so this is not testing re-draw.

Reply:
Clearly I need to improve the name. The test is doing what I wanted it to do (making multiple figures), so will fix up the name/description.